client/package.json

```diff
@@ -44,8 +44,45 @@
     },
     "extends": [
       "react-app",
-      "eslint:recommended"
-    ]
+      "eslint:recommended",
+      "plugin:react/recommended"
+    ],
+    "plugins": [
+      "react",
+      "import"
+    ],
+    "rules": {
+      "array-bracket-newline": "warn",
+      "array-bracket-spacing": "warn",
+      "camelcase": "warn",
+      "eqeqeq": "error",
+      "no-duplicate-imports": "error",
+      "no-else-return": "error",
+      "no-unneeded-ternary": "error",
+      "no-useless-rename": "error",
+      "prefer-arrow-callback": "error",
+      "prefer-destructuring": [
+        "error",
+        {
+          "object": true,
+          "array": false
+        }
+      ],
+      "import/order": [
+        "warn",
+        {
+          "alphabetize": {
```

Comment:
is it useful?

Reply:
I think so, for files with a lot of imports, it allows to find the one you search directly.
But if you think it's an extra overhead, I can remove it.